docs-ref-conceptual/node-vscode-deployment-zip.yml

```diff
@@ -0,0 +1,36 @@
+### YamlMime:YamlDocument
+documentType: Tutorial
+title: Deploy Node.js Application to Azure App Service with Visual Studio Code
+metadata:
+  title: Deploy Node.js Application to Azure App Service with Visual Studio Code
+  description: Use Visual Studio Code extensions for Docker to deploy a Node.js application to Azure App Service
+  audience: Developer
+  level: Intermediate
+  nextTutorialHref: unknown
+  nextTutorialTitle: ** TODO **
+  displayType: one-column
+  interactive: azurecli
+items:
+- durationInMinutes: 5
+  content: |
+    This tutorial demonstrates how to deploy a Node.js application to Azure App Service using [Visual Studio Code extensions](https://code.visualstudio.com/docs/editor/extension-gallery).
+
+    Before you begin, make sure you have the following items:
+
+    - [Azure account](https://azure.microsoft.com/free/)
+    - [Visual Studio Code](https://code.visualstudio.com/download)
+    - [Docker](https://www.docker.com/)
+    - [Node Pack for Azure](https://marketplace.visualstudio.com/items?itemName=ms-vscode.vscode-node-azure-pack) for Visual Studio Code
+- title: Create a new application
+  durationInMinutes: 2
+  content: |
+    From the Visual Studio Code **App Service** extension, click the `+` button to create a new application. Once clicked, the editor displays a series of prompts at the top of the screen. 
+
+    > [!NOTE]
+    > Note: To observe the status as Azure creates your application, you can open Visual Studio Code's Integrated Terminal through the `View > Integrated Terminal` menu option.
```

Comment:
I haven't seen one of these interactive tutorials rendered, but "Note" is added the note box header automatically on docs.ms.com. As such, recommend striking "Note:" from this note.